src/shared/cloudformation/templateRegistry.ts

```diff
@@ -40,12 +40,15 @@ export class CloudFormationTemplateRegistry {
      * Adds template to registry. Wipes any existing template in its place with newly-parsed copy of the data.
      * @param templateUri vscode.Uri containing the template to load in
      */
-    public async addTemplateToRegistry(templateUri: vscode.Uri): Promise<void> {
+    public async addTemplateToRegistry(templateUri: vscode.Uri, quiet?: boolean): Promise<void> {
         const pathAsString = templateUri.fsPath
         try {
             const template = await CloudFormation.load(pathAsString)
             this.templateRegistryData.set(pathAsString, template)
         } catch (e) {
+            if (!quiet) {
```

Comment:
This change appears to introduce throws into existing code. Will this cause unexpected behavior?
Consider unconditionally making the log call on line 52. (also, it looks like line 52 should be passing e as a second arg to `verbose`. The logger handles formatting and error output.

Reply:
this actually restores old behavior, to make some tests happy. Might revisit later